### PR TITLE
Proxies

### PR DESCRIPTION
**Pull request by dwclark**

### CODE REVIEW

File: http-builder-ng-core/src/main/java/groovyx/net/http/ChainedHttpConfig.java [hunk 1]
```diff
@@ -161,7 +161,7 @@ default Object actualContext(final String contentType, final Object id) {
 
     /**
      * Used to find the encoder configured to encode the specified content-type.
-     *
+     *[
```
Comment (cjstehno) [inline http-builder-ng-core/src/main/java/groovyx/net/http/ChainedHttpConfig.java]: Typo?

**Comment by cjstehno** [comment]
That was a hell of a lot more work than I thought it was going to be... geesh. Are you going to add anything to the user guide or do you want me to do that?

File: http-builder-ng-core/src/main/java/groovyx/net/http/ChainedHttpConfig.java (hunk 1 above)
Comment (dwclark) [reply http-builder-ng-core/src/main/java/groovyx/net/http/ChainedHttpConfig.java]: Yeah, stray emacs meta command.

**Comment by dwclark** [comment]
I just added a Proxying section to the user guide. I think that covers everything. Let me know if there are other issues or typos you see.

**Comment by cjstehno** [comment]
Do you want to get more stuff into a release or just cut this ASAP?

**Comment by dwclark** [comment]
Let me try and get the removal of the type checking config script in as well. If I don't have it finished by tomorrow, let's release then.

**Comment by dwclark** [comment]
OK, I removed the typecheckhttpconfig.groovy and all the code that utilized it. Let's go ahead and release.

**Comment by cjstehno** [comment]
Ok, I will release tomorrow morning.

**Pull request merged**